tests/de.ovgu.featureide.fm.ui-test/pom.xml

```diff
@@ -1,6 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+	<repositories>
```

Comment:
I would prefer to integrate SWT in a separate issue/PR. Would that be feasible?

Reply:
We have removed the testing code and will add it in a separate Pull Request/Branch.